src/components/markdown-renderer/replace-components/image/image-frame.tsx

```diff
@@ -3,7 +3,7 @@ import { useSelector } from 'react-redux'
 import { getProxiedUrl } from '../../../../api/media'
 import { ApplicationState } from '../../../../redux'
 
-export const ImageFrame: React.FC<React.ImgHTMLAttributes<HTMLImageElement>> = ({ alt, src, ...props }) => {
+export const ImageFrame: React.FC<React.ImgHTMLAttributes<HTMLImageElement>> = ({ alt, src, title, ...props }) => {
```

Comment:
Is this necessary? The title prop is covered by `...props`

Reply:
The same is true for `alt`? I wanted to make it obvious that the title get's set.
To answer the questions: No, it is not.

Reply:
The alt tag is explicitly set, because otherwise the linter is screaming, because it should be set.

Reply:
I get your point. If you want to set the title explicitly, too, then it's fine for me.

Reply:
then would you resolve this an review again?